SecurityPkg/Library/DxeImageVerificationLib/DxeImageVerificationLib.c

```diff
@@ -2049,8 +2049,9 @@ DxeImageVerificationHandler (
   // executable information table in either case.
   //
   NameStr = ConvertDevicePathToText (File, FALSE, TRUE);
-  AddImageExeInfo (Action, NameStr, File, SignatureList, SignatureListSize);
+
   if (NameStr != NULL) {
+    AddImageExeInfo (Action, NameStr, File, SignatureList, SignatureListSize);

```

Comment:
Should this function still be called even if the devicepath to text fails?  Seems like it might be important to add even in case the name fails.